helper/include/odp/helper/cli.h

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2021, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * ODP CLI helper API
```

Comment:
This header should be included in `helper/include/odp/helper/odph_api.h`. You can wrap the include with e.g. `#ifdef _ODPH_CLI` (see `_ODP_PKTIO_NETMAP` for example).

Reply:
_ODP_PKTIO_NETMAP is probably not a good template here, since it's in an ODP autoheader. Helper doesn't have an autoheader. Should we create an autoheader for ODP helper?